class Break_(Exception):
    pass
